test/files/composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "test/sample-composer-project",
```

Comment:
Right. I was trying to avoid an extra argument in the specs, but I can add it. Some managers can't retrieve the project name anyways, so it's going to be cleaner.